include/boost/geometry/policies/robustness/segment_ratio.hpp

```diff
@@ -254,21 +262,14 @@ public :
         return m_numerator > m_denominator;
     }
 
-    inline bool near_end() const
-    {
-        if (left() || right())
-        {
-            return false;
-        }
-
-        static fp_type const small_part_of_scale = scale() / 100;
-        return m_approximation < small_part_of_scale
-            || m_approximation > scale() - small_part_of_scale;
-    }
-
-    inline bool close_to(thistype const& other) const
+    //! Returns a value between 0.0 and 1.0
+    //! 0.0 means: exactly in the middle
+    //! 1.0 means: exactly on one of the edges (or even over it)
+    inline floating_point_type edge_value() const
     {
-        return geometry::math::abs(m_approximation - other.m_approximation) < 50;
+        floating_point_type const result
+                = 2.0 * std::fabs(0.5 - m_approximation / scale());
```

Comment:
`geometry::math::abs` should probably still be used here because `std::fabs` may be not specialized for user-defined FP type. Also because it is used below too, so for consistency.

Reply:
:heavy_check_mark: